examples/apollo-watch-fragments/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@apollo/client": "~3.6.0",
+    "@apollo/client": "3.8.1",
```

Comment:
Is this the version we are using in TMP today? If not, I believe we agreed to use TMP's version and include a patch-package.

Reply:
nope, TMP uses 3.6.10. I gonna leave the graphitation as it is and patch it

Reply:
because dep-up duct-tape's Apollo will cause problem in TMP as it is patched there. Discussed w/@vladar so far the easiest way is to patch duct-tape's Apollo (btw it is already it is), and only dep-up duct-tape in TMP.